tsz-compress/src/v2/halfvec.rs

```diff
@@ -185,6 +185,25 @@ impl HalfVec {
     }
 }
 
+/// Compares two HalfVec for equality.
+impl PartialEq for HalfVec {
+    fn eq(&self, other: &Self) -> bool {
+        self.words == other.words && self.len == other.len
+    }
+}
+
+/// Compares two HalfWord for equality.
+impl PartialEq for HalfWord {
```

Comment:
Similar to above, we do not have a use case for this within the library as far as I know.

Reply:
Changes in commit [c19be9f](https://github.com/qsib-cbie/tsz/pull/36/commits/c19be9f42ebca7fe539b84512e03f6fb671f51db).